Dockerfile

```diff
@@ -31,7 +31,10 @@ COPY dist /app/
 CMD [ "/app/custom-pod-autoscaler" ]
 
 # Alpine build
-FROM alpine:3.10 AS alpine
+FROM registry.c3.ai/ubi/ubi8-minimal:8.3.230.2 AS c3-ubi
```

Comment:
New one is available - https://c3energy.atlassian.net/wiki/spaces/OPS/pages/2303625488/UBI+8

Reply:
done.